### ISSUE TITLE
🦠 Model Request: Normalized WHALES

### ISSUE THREAD

**Issue by miquelduranfrigola** [ersilia-os/ersilia#986]
### Model Name

WHALES scaled

### Model Description

Scaled version of WHALES descriptors. WHALES are holistic molecular descriptors useful for scaffold hopping.

### Slug

whales-scaled

### Tag

Descriptor

### Publication

https://www.nature.com/articles/s42004-018-0043-x

### Source Code

https://github.com/grisoniFr/scaffold_hopping_whales

### License

MIT

**Comment by miquelduranfrigola** [comment]
This model can build upon the previous WHALES model, with scaling. I would apply Robust scaling from sklearn. If installing sklearn is problematic, we can do it explicitly with median etc.

**Comment by GemmaTuron** [comment]
This model is now incorporated.
